students/rhamersky/Lesson_10/mailroom_part6.py

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env python3
+# Description: This program will allow a user to input a donor and a donation.
+# The user will also be able to send out thank you notes to the donors.
+# Developer: Ryan Hamersky
+# Date: 05/21/2018
+# Rev: New
+
+# -----Data Section-----
+str_name = str()    # --> User's name
+int_user_input = int()  # --> User's option.
+dic_donors_sorted = {}  # --> Sorted dictionary of donors.
+dic_donors = {"William Gates, III": [10000.0, 1], "Mark Zuckerberg": [20000.0, 1],
+                   "Jeff Bezos": [7800.0,4, 2500, 2000, 3000, 300],"Paul Allen": [5000.0,1],
+                   "Mike Scott": [25.0,1]} # --> Dic that stores all donor's information
+str_salutation = ""   # --> Email salutation
+str_body = ""         # --> Email body
+str_valediction = ""  # --> Email valediction
+
+
+# -----Process Section-----
+# This class creates an donor object.
+class Donor:
+    initial_donation = 1  # --> Field is used as the initial donation is constant with all donor objects.
+
+    def __init__(self, donor, donation):  # --> Constructs attributes donor and donation.
+        self.donor = donor
+        self.donation = donation
+
+########### END OF DONOR OBJECT CLASS###############
+
+# Methods in the Menu class will execute code based off of the user's menu choice.
+
+
+class Menu:
```

Comment:
In Python you would generally define these methods as module level functions rather than introducing the extra overhead of using a class with only static methods